bin/http-server

```diff
@@ -48,11 +53,39 @@ var port = argv.p || parseInt(process.env.PORT, 10),
     ssl = !!argv.S || !!argv.ssl,
     proxy = argv.P || argv.proxy,
     utc = argv.U || argv.utc,
+    logfile = (function () {
```

Comment:
Instead of a messy IIFE, could we put this checking into its own function, and call it here?